keps/sig-storage/20191125-bucket-provisioning.md

```diff
@@ -274,8 +272,8 @@ status:
 1. `bucketClassName`: Name of the associated `BucketClass`.
 1. `supportedProtocols`:  String array of protocols (e.g. s3, gcs, swift, etc.) supported by the associated object store.
 1. `releasePolicy`: the release policy defined in the associated BucketClass (see [BucketClass](#BucketClass) for more information).
-1. `bucketRef`: the name & namespace of the bound `Bucket`.
-1. `secretName`: the name of the app secret in the `Bucket`'s namespace, as provided in `bucketRef.namespace`.
+1. `bucketRef`: the name & namespace of the associated `Bucket`. For both brownfield cases, `bucketRef` is set by an admin and names the bucket to be accessed. For greenfield, it is added by the COSI controller.
```

Comment:
In Brownfield, the COSI controller can still manage binding operations.  I know that pre-binding was a thing in k8s before dynamic provisioning, but I think in hindsight, it wasn't regarded as a great solution.  Binding and secret copying, under brownfield, can be completely automated.

Reply:
Agree since I assume by _binding_ I think we mean that `Bucket.status.bucketContentName` == `BucketContent.name` && `BucketContent.bucketRef.name` ==  `Bucket.name`

Reply:
Right, that's part of it, but those are effectively meaningless if the Secret isn't provided to the user via automation.